site/class-1-bar/data.csv

```diff
@@ -0,0 +1,24 @@
+"date","steps";
```

Comment:
This looks great, but you’re uploading files to `site/class-1-bar` instead of `site/class-1-bar/nickanthonygroen`!